tools/Google.Events.Tools.CodeGenerator/Program.cs

```diff
@@ -125,6 +125,45 @@ private static void GenerateCode(IEnumerable<CloudEventDataInfo> infos, string s
             }
         }
 
+        private static void GenerateCloudEventAttributes(FileDescriptorSet descriptorSet, string srcDirectory, string project)
+        {
+            // TODO: Use the information about which extension attribute is used within which event type?
```

Comment:
If you want to translate this to something static, say `{EventType}ExtensionAttributes` classes, that'd be a breaking change. Unless you are thinking more about validation?
I think that both static and validation would be nice.

Reply:
I didn't have a specific scenario in mind - but I think documentation generation would be a good start, as described above.

Reply:
Yep documentation sounds good actually.